contrib/crio-wipe/crio-wipe.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
```

Comment:
Let's set -u and set -e. Also, any reason to not just use bash?

Reply:
...not sure why I set it as sh